Source/WebInspectorUI/UserInterface/Views/ConsoleSession.js

```diff
@@ -46,7 +46,7 @@ WI.ConsoleSession = class ConsoleSession extends WI.Object
             headerText = WI.UIString("Page reloaded at %s");
             break;
         case WI.ConsoleSession.NewSessionReason.PageNavigated:
-            headerText = WI.UIString("Page navigated at %s");
+            headerText = WI.UIString("Navigated to %s").format(WI.networkManager.mainFrame.url);
```

Comment:
we should also probably call something like `WI.truncateURL(...)` or `.truncateMiddle(100)` to ensure that it doesnt show the entire URL if it's super long